src/core/client/admin/components/UserStatus/SuspendModal.tsx

```diff
@@ -0,0 +1,195 @@
+import { Localized } from "fluent-react/compat";
+
+import { Format, withFormat } from "coral-framework/lib/i18n";
+import React, { FunctionComponent, useState } from "react";
+
+import NotAvailable from "coral-admin/components/NotAvailable";
+import {
+  Button,
+  Card,
+  CardCloseButton,
+  CheckBox,
+  Flex,
+  HorizontalGutter,
+  Modal,
+  RadioButton,
+  Typography,
+} from "coral-ui/components";
+import SuspendSuccessModal from "./SuspendSuccessModal";
+
+import styles from "./SuspendModal.css";
+
+interface Props {
+  username: string | null;
+  open: boolean;
+  onClose: () => void;
+  onSuccessClose: () => void;
+  onConfirm: (timeout: number, message: string) => void;
+  format: Format;
+  organizationName: string;
+  success: boolean;
+}
+
+const DURATIONS: Array<[string, string]> = [
+  ["3600", "1 hour"],
+  ["10800", "3 hours"],
+  ["86400", "24 hours"],
+  ["604800", "7 days"],
+];
```

Comment:
There is a DurationField under `src/core/client/framework/components/DurationField.tsx` that shows a way to do duration that returns localized values.

As well, @wyattjoh nicely refactored my implementation of similar using neat templating in another PR: https://github.com/coralproject/talk/pull/2415/commits/9ee56594dfdcee5610390c2bd4a008bd4fa13c08

It might be useful to structure this similarly.